content/blog/2024-07-07-aws-to-contribute-efficiency-improvements-for-ValKey-8.md

```diff
@@ -0,0 +1,82 @@
++++
+# `title` is how your post will be listed and what will appear at the top of the post
+title= "Unlock 1 Million RPS:  Experience Triple the Speed with Valkey 8.0 "
+# `date` is when your post will be published.
+# For the most part, you can leave this as the day you _started_ the post.
+# The maintainers will update this value before publishing
+# The time is generally irrelvant in how Valkey published, so '01:01:01' is a good placeholder
+date= 2024-07-08 01:01:01
+# 'description' is what is shown as a snippet/summary in various contexts.
+# You can make this the first few lines of the post or (better) a hook for readers.
+# Aim for 2 short sentences.
+description= "Better efficiency reduces cost, improves latency and makes our environment greener"
+# 'authors' are the folks who wrote or contributed to the post.
+# Each author corresponds to a biography file (more info later in this document)
+authors= [ "dantouitou", "uriyagelnik"]
++++
+Valkey, the popular open-source key-value store that succeeded OSS Redis, captures the imagination of developers around the world.
+From simple in-memory caching implementations to complex job queues, real-time collaboration, and leaderboard applications, we at AWS are continually amazed by how innovatively users employ Valkey.
+Clearly, this is just the tip of the iceberg.
+As more use cases and industries seek to benefit from the speed, low latency, and cost reduction advantages of in-memory processing as introduced by Valkey, we are fully committed to our vision of delivering exceptional performance.
+
+### Our Commitment to performance and efficiency
+At AWS, we have hundreds of thousands of customers using Amazon ElastiCache and Amazon MemoryDB.
+Feedback we continuously hear from customers is that they need better absolute performance and squeeze more performance from their clusters.
+
+For absolute performance per node, customers sometimes have hot keys or other limitation that prevents them from scaling out.
+They also need better latencies at high percentiles.
+For efficiency, we see customers trying to optimize their costs and get more out of their boxes.
+
+Besides performance, we also see efficiency as a way to improve resilience.
+For example, with fork-based snapshotting and full-sync replication, the longer it takes, the more memory is consumed by copied-on-write pages and the replica output buffer, often resulting in high swapping and erratic behavior.
+By reducing execution time, one can achieve a more predictable memory usage and greater stability.
```

Comment:
I have another comment suggesting we move `### Benefits of High Capacity Shards` here. If you take that, I think this text should moved into that paragraph.

Reply:
Done